python/lsst/cp/pipe/ptc/cpSolvePtcTask.py

```diff
@@ -187,10 +187,11 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
             Output data refs to persist.
         """
         inputs = butlerQC.get(inputRefs)
-        outputs = self.run(inputCovariances=inputs['inputCovariances'], camera=inputs['camera'])
+        detId = inputRefs.inputCovariances[0].dataId['detector']
+        outputs = self.run(inputCovariances=inputs['inputCovariances'], camera=inputs['camera'], detId=detId)
```

Comment:
Doesn't this need to pass `inputExpList`?  It didn't previously it seems, but doesn't that mean that it's always `None`?

Reply:
Was it only being used to find a detector before, and now that's being handled by the dimension information in `runQuantum`, so it isn't needed anymore?

Reply:
I don't think it was used before (except for that one check, but then it was always ``None``), so I removed it.